packages/slice-machine/components/AppLayout/Navigation/ChangesListItem.tsx

```diff
@@ -67,12 +71,19 @@ export const ChangesListItem = forwardRef<HTMLLIElement, ChangesListItemProps>(
 );
 
 const useOpenChangesHoverCard = () => {
```

Comment:
This is good, it does what it needs to do.

What I'm wondering though is should we make this generic enough to be reused by the other or future hover cards by returning all the boolean properties in an object?

Reply:
Hum maybe, but for now this is the first and only tooltip that wait for something to be displayed. Other one are just if there is false in LS, display it.

So I guess it would be hard to make it generic now. WDYT?

Reply:
I think if we went that way it would mostly look like a wrapper or util for getting the `hasSeen...` from redux. Not need for now so I'll resolve this one :)